examples/reg.py

```diff
@@ -283,6 +284,8 @@ def add(self, dce, keyName):
                 rrp.REG_QWORD, rrp.REG_QWORD_LITTLE_ENDIAN
             ):
                 valueData = int(self.__options.vd)
+            elif dwType == rrp.REG_BINARY:
+                valueData = binascii.a2b_hex(self.__options.vd)
```

Comment:
Something like this...

```suggestion
            elif dwType == rrp.REG_BINARY:
                bin_value_len = len(self.__options.vd)
                bin_value_len += (bin_value_len & 1)
                valueData = binascii.a2b_hex(self.__options.vd.ljust(bin_value_len, '0'))
```

Reply:
Great , should never trust user's input.